### DISCUSSION TITLE
编译时出错：Unable to find package manifest for 'StarcoinFramework'

### DISCUSSION THREAD

**Discussion by ChenpengZhang** [NonceGeek/Move-Camp#148] [Q&A] [answered]
按照教程写好了最简单的结构，但是却显示不能编译......
我真的纯小白，这大概是个挺低级的错误，但是我实在找不到有什么原因，是没有下载某些dependencie么
<img width="1315" alt="Screen Shot 2022-10-23 at 14 22 55" src="https://user-images.githubusercontent.com/96060895/197377379-8757f551-6b4b-4671-9052-5a3c4b5c5108.png">

<img width="901" alt="Screen Shot 2022-10-23 at 14 27 00" src="https://user-images.githubusercontent.com/96060895/197377493-daa8de4f-750b-43e0-94f1-a3334b7e3ecc.png">

**Comment by geometryolife** [comment, marked as answer]
这个可能是因为网络问题，没能成功下载 StarcoinFramework，尝试挂个代理。

**Comment by Demian101** [comment]
看起来是安装出问题了，但是不能确定是哪一步，
我写一个 StarCoin 的详细 setup，你不妨照着再操作一遍  🆙
[StarCoin Setup.pdf](https://github.com/NonceGeek/Web3-dApp-Camp/files/9845919/StarCoin.Setup.pdf)
